core/modules/query/WhereClause.h

```diff
@@ -32,30 +32,30 @@
   * @author Daniel L. Wang, SLAC
   */
 
+
 // System headers
 #include <iostream>
 #include <memory>
 #include <stack>
+#include <vector>
 
-// Local headers
-#include "query/BoolTerm.h"
-#include "query/ColumnRef.h"
-#include "query/QsRestrictor.h"
 
-// Forward declarations
 namespace lsst {
 namespace qserv {
 namespace parser {
     class WhereFactory;
```

Comment:
There shouldn't be any indentation before the forward declarations (or anything within namespaces)